{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -168,12 +115,10 @@ exclude = [
     "**/settings/*",
     "snapshots",
 ]
-max-arguments = 12
```

Comment:
Duplicated in Pylint - `max-args`

Reply:
Yes, but flake8 will raise an error by this rule. You may check it localy

Reply:
`max-arguments` config option is available for [wemake](https://wemake-python-styleguide.readthedocs.io/en/latest/pages/usage/configuration.html) or [flake8-functions](https://github.com/best-doctor/flake8-functions?tab=readme-ov-file#flake8-functions). It is not default config option - [doc search](https://flake8.pycqa.org/en/latest/search.html?q=max-arguments&check_keywords=yes&area=default#)